package.json

```diff
@@ -45,8 +45,7 @@
     "add-textdomain": "wpi18n addtextdomain --textdomain=jetpack --glob-pattern='!(docker|node_modules|tests|tools|vendor){*.php,**/*.php}'",
     "test-gui": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js gui",
     "precommit": "node bin/pre-commit-hook.js",
-    "test-modules": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js modules",
-    "test-dangerci-and-adminpage": "yarn danger ci && yarn test-adminpage"
```

Comment:
I didn't see this `test-dangerci-and-adminpage` being used anywhere else except in Travis scripts so for the sake of simplicity, I've removed it and I'm running those two tasks directly instead.